appstudio-utils/util-scripts/lib/fetch/rekor.sh

```diff
@@ -79,7 +79,7 @@ rekor-log-entry-save-from-url() {
 shorten-sha() {
   local sha=$1
 
-  echo "$sha" | head -c 11
+  echo "$sha" | sed 's|^sha256:||' | head -c 11
```

Comment:
Might not be that important, but in case we switch to sha384/sha512 might make it future proof

```suggestion
  echo "$sha" | sed 's|^.+:||' | head -c 11
```

Reply:
I did a version of that in new revision, along with some (overly enthuastic?) TDD since I don't trust my sed regex abilities.